pkg/scheduler/backend/cache/cache_test.go

```diff
@@ -338,6 +339,7 @@ func TestExpirePod(t *testing.T) {
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
 			logger, ctx := ktesting.NewTestContext(t)
+			metrics.Register()
```

Comment:
Can we have a one consistent place where we're calling the `metrics.Register`, e.g. always after `defer cancel()`? Right now, they are mixed.

Reply:
Done

Reply:
Sorry missed this one, but yes you're right we could just move it to be done once at the top level for each test instead of registering for each test-case. Have made those changes now.